### PR TITLE
Add documentation about TextControl

### PR DESCRIPTION
**Pull request by SebastianNiehusAA**
# Description
PHS-729

## Before Merging
 - [x] Review the code changes
    - Unused print / comments / TODOs
    - Missing docstrings for functions that should have them
    - Consistent variable names
    - ...
 - [x] Update `changelog.md` if necessary
 - [x] Commit messages should contain a semantic [label](https://gist.github.com/joshbuchea/6f47e86d2510bce28f8e7f42ae84c716) and the ticket number
   - Consider squashing if this is not the case

### CODE REVIEW

File: src/documentation/attention_manipulation_with_text_controls.ipynb [hunk 1]
```diff
@@ -0,0 +1,286 @@
...
+    "\n",
+    "\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "2",
+   "metadata": {},
+   "source": [
+    "`TextControls` enable us to increase or decrease the attention of our model on different parts of the prompt (attention manipulation, AtMan).\n",
```
Comment (NiklasKoehneckeAA) [inline src/documentation/attention_manipulation_with_text_controls.ipynb:33]:
```suggestion
    "`TextControl`s enable us to increase or decrease the attention of our model on different parts of the prompt (attention manipulation, AtMan).\n",
```
maybe? never a big fan of this but not sure how to do this better

**Review by NiklasKoehneckeAA — COMMENTED** [review]
I think the tutorial itself is fine. However, it is risky to refer to the specific output of the models within the tutorials, we ran into some issues with this before. The behavior may change without us noticing, and then the tutorial makes no sense anymore. Maybe we should take this into consideration. However, I think its fine for now and "good enough". One quick fix may be to include what the output was that we got so people can double check if what they get was what we got when we wrote this

File: src/documentation/attention_manipulation_with_text_controls.ipynb (hunk 1 above)
Comment (SebastianNiehusAA) [reply src/documentation/attention_manipulation_with_text_controls.ipynb:33]: True

**Comment by SebastianNiehusAA** [comment]
> I think the tutorial itself is fine. However, it is risky to refer to the specific output of the models within the tutorials, we ran into some issues with this before. The behavior may change without us noticing, and then the tutorial makes no sense anymore. Maybe we should take this into consideration. However, I think its fine for now and "good enough". One quick fix may be to include what the output was that we got so people can double check if what they get was what we got when we wrote this

Good point. Added.

**Pull request merged**